test-specifications/cryptoService/accountCreateTransaction.md

```diff
@@ -43,19 +43,21 @@ const transaction = new AccountCreateTransaction()
 
 ### Parameters
 
-| Parameter Name            | Type   | Required/Optional | Description/Notes                                                                                                                           |
-|---------------------------|--------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
-| key                       | string | optional          | DER-encoded hex string representation for private or public keys. Keylists and threshold keys are the hex of the serialized protobuf bytes. |
-| initialBalance            | int64  | optional          | Units of tinybars                                                                                                                           |
-| receiverSignatureRequired | bool   | optional          |                                                                                                                                             |
-| autoRenewPeriod           | int64  | optional          | Units of seconds                                                                                                                            |
-| memo                      | string | optional          |                                                                                                                                             |
-| maxAutoTokenAssociations  | int32  | optional          |                                                                                                                                             |
-| stakedAccountId           | string | optional          |                                                                                                                                             |
-| stakedNodeId              | int64  | optional          |                                                                                                                                             |
-| declineStakingReward      | bool   | optional          |                                                                                                                                             |
-| alias                     | string | optional          | Hex string representation of the keccak-256 hash of an ECDSAsecp256k1 public key type.                                                      |
-| signerKey                 | string | optional          | DER-encoded hex string representation of an additional private key required to sign.                                                        |
+| Parameter Name            | Type   | Required/Optional | Input/Output | Description/Notes                                                                                                                           |
```

Comment:
I think its better to have 2 tables: one for input parameters and the other for output.

Reply:
Split out parameters into input and output tables

Reply:
Great. Also I think that 'Required/Optional' column can be removed from the table 'Output Parameters'. Wdyt?

Reply:
Sure, I suppose I can't think of any case where there would be optional return data.

Reply:
Removed